lib/utils.rb

```diff
@@ -88,7 +88,7 @@ def get_release_info(author, repository, options)
   Puppet.debug('Parsing release info')
 
   # noinspection RubyResolve
-  PSON.load(release_info_json)
+  JSON.load(release_info_json)
```

Comment:
[Rubocop](https://github.com/dodevops/puppet-githubreleases/actions/runs/8858862187/job/24327994868?pr=8) complains about using JSON.parse instead of JSON.load. Could you kindly change that?

Reply:
Done